webservices/resources/large_aggregates.py

```diff
@@ -0,0 +1,60 @@
+from flask_apispec import doc
+
+from webservices import args
+from webservices import docs
+from webservices import utils
+from webservices import schemas
+from webservices.common import models
+from webservices.common.views import ApiResource
+
+
+@doc(
+    tags=['financial'],
+    description=docs.ENTITY_RECEIPTS_TOTLAS,
+)
+class EntityReceiptsTotalsView(ApiResource):
+
+    model = models.EntityReceiptsTotals
+    schema = schemas.EntityReceiptsTotalsSchema
+    page_schema = schemas.EntityReceiptsTotalsPageSchema
+
+    filter_match_fields = [
+        ('cycle', model.cycle),
+    ]
+
+    @property
+    def args(self):
+        return utils.extend(
+            args.paging,
+            args.large_aggregates,
+        )
+
+    @property
+    def index_column(self):
+        return self.model.idx
+
+
+@doc(
+    tags=['financial'],
+    description=docs.ENTITY_DISBURSEMENTS_TOTLAS,
+)
+class EntityDisbursementsTotalsView(ApiResource):
```

Comment:
Is there any reason that these were broken out into two views?  If anything is expected to change with these endpoints (e.g. adding a new filter) it could be problematic as the changes will have to made to both views as opposed to using just one; we do this on the `reports\` endpoint by injecting the model, schema, and page schema at runtime based on what endpoint was requested.